packages/crdt/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@actual-app/crdt",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "CRDT layer of Actual",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "lint": "eslint .",
+    "build:node": "tsc --p tsconfig.dist.json",
+    "build": "rm -rf dist && yarn run build:node && cp src/proto/sync_pb.d.ts dist/src/proto/",
```

Comment:
I'm not super happy with the last manual `cp` step, but I'm out of ideas how to make this work properly in the short-term.